### PR TITLE
Update baseline-status.js

### PR DESCRIPTION
**Pull request by rachelandrew**
Changing the Chrome SVG for a correct one.

Fixes: #12

### CODE REVIEW

**Comment by devnook** [comment]
The new logo does not scale well as it is now
Screenshot 2024-10-31 at 21 07 56

I'll take a look tomorrow on how to tweak it to the right size.

**Comment by bramus** [comment]
@devnook: @mathiasbynens has a version that scales well to all possible sizes. Can you add Ewa to the repo, Mathias?

File: baseline-status.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Task } from '@lit/task';
...
+  chrome: html`<svg xmlns="http://www.w3.org/2000/svg" viewBox="-10 -10 276 276"><linearGradient id="a" x1="145" x2="34" y1="253" y2="61" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#1e8e3e"/><stop offset="1" stop-color="#34a853"/></linearGradient><linearGradient id="b" x1="111" x2="222" y1="254" y2="62" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fcc934"/><stop offset="1" stop-color="#fbbc04"/></linearGradient><linearGradient id="c" x1="17" x2="239" y1="80" y2="80" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#d93025"/><stop offset="1" stop-color="#ea4335"/></linearGradient><circle cx="128" cy="128" r="64" fill="#fff"/><path fill="url(#a)" d="M96 183.4A63.7 63.7 0 0 1 72.6 160L17.2 64A128 128 0 0 0 128 256l55.4-96A64 64 0 0 1 96 183.4Z"/><path fill="url(#b)" d="M192 128a63.7 63.7 0 0 1-8.6 32L128 256A128 128 0 0 0 238.9 64h-111a64 64 0 0 1 64 64Z"/><circle cx="128" cy="128" r="52" fill="#1a73e8"/><path fill="url(#c)" d="M96 72.6a63.7 63.7 0 0 1 32-8.6h110.8a128 128 0 0 0-221.7 0l55.5 96A64 64 0 0 1 96 72.6Z"/></svg>`,
```
Comment (devnook) [inline baseline-status.js:7]:
```suggestion
  chrome: html`<svg xmlns="http://www.w3.org/2000/svg" width="21" height="21" viewBox="0 0 260 260"><linearGradient id="a" x1="145" x2="34" y1="253" y2="61" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#1e8e3e"/><stop offset="1" stop-color="#34a853"/></linearGradient><linearGradient id="b" x1="111" x2="222" y1="254" y2="62" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fcc934"/><stop offset="1" stop-color="#fbbc04"/></linearGradient><linearGradient id="c" x1="17" x2="239" y1="80" y2="80" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#d93025"/><stop offset="1" stop-color="#ea4335"/></linearGradient><circle cx="128" cy="128" r="64" fill="#fff"/><path fill="url(#a)" d="M96 183.4A63.7 63.7 0 0 1 72.6 160L17.2 64A128 128 0 0 0 128 256l55.4-96A64 64 0 0 1 96 183.4Z"/><path fill="url(#b)" d="M192 128a63.7 63.7 0 0 1-8.6 32L128 256A128 128 0 0 0 238.9 64h-111a64 64 0 0 1 64 64Z"/><circle cx="128" cy="128" r="52" fill="#1a73e8"/><path fill="url(#c)" d="M96 72.6a63.7 63.7 0 0 1 32-8.6h110.8a128 128 0 0 0-221.7 0l55.5 96A64 64 0 0 1 96 72.6Z"/></svg>`,
```

**Comment by devnook** [comment]
@rachelandrew I cannot edit your fork, so I added a suggestion to fix the icon in the PR. If you apply it I can then merge the PR.

**Comment by rachelandrew** [comment]
@devnook done

**Comment by mathiasbynens** [comment]
> @devnook: @mathiasbynens has a version that scales well to all possible sizes. Can you add Ewa to the repo, Mathias?

Done! Here’s the relevant code:

```svg
<svg width="256" height="256" fill="none" xmlns="http://www.w3.org/2000/svg"><style>.conditional{display:none}@media (min-width:1px) and (max-width:20px){#g-1-20{display:block}}@media (min-width:21px) and (max-width:127px){#g-21-127{display:block}}@media (min-width:128px){#g-128-plus{display:block}}</style><g class="conditional" id="g-1-20"><g clip-path="url(#clip-path-1-20)"><path d="M128 191.916c35.346 0 64-28.654 64-64 0-35.346-28.654-64-64-64-35.346 0-64 28.654-64 64 0 35.346 28.654 64 64 64Z" fill="#fff"/><path d="M159.96 72.468a64.01 64.01 0 0 1 23.432 87.446l.029.016L128 255.916a127.973 127.973 0 0 0 110.847-63.992 127.967 127.967 0 0 0-.023-127.992H130.269a63.706 63.706 0 0 1 29.691 8.536Z" fill="url(#linear-gradient-1-20-a)"/><path d="M128 175.916c26.51 0 48-21.49 48-48s-21.49-48-48-48-48 21.49-48 48 21.49 48 48 48Z" fill="#1A73E8"/><path d="M63.984 127.964A64.016 64.016 0 0 1 128 63.948v-.034h110.833a127.968 127.968 0 0 0-221.675.016l54.279 94.015a63.71 63.71 0 0 1-7.453-29.981Z" fill="url(#linear-gradient-1-20-b)"/><path d="M160.056 183.346a64.012 64.012 0 0 1-70.979-4.651 64.016 64.016 0 0 1-16.469-18.78l-.029.016-55.415-95.984a127.968 127.968 0 0 0 110.851 191.968l54.28-94.014a63.728 63.728 0 0 1-22.239 21.445Z" fill="url(#linear-gradient-1-20-c)"/></g><defs><linearGradient id="linear-gradient-1-20-a" x1="223.279" y1="60.863" x2="112.442" y2="252.839" gradientUnits="userSpaceOnUse"><stop stop-color="#FBBC04"/><stop offset="1" stop-color="#FCC934"/></linearGradient><linearGradient id="linear-gradient-1-20-b" x1="17.158" y1="78.929" x2="238.833" y2="78.929" gradientUnits="userSpaceOnUse"><stop stop-color="#D93025"/><stop offset="1" stop-color="#EA4335"/></linearGradient><linearGradient id="linear-gradient-1-20-c" x1="141.004" y1="248.416" x2="30.167" y2="56.44" gradientUnits="userSpaceOnUse"><stop stop-color="#1E8E3E"/><stop offset="1" stop-color="#34A853"/></linearGradient><clipPath id="clip-path-1-20"><path fill="#fff" d="M0 0h256v256H0z"/></clipPath></defs></g><g class="conditional" id="g-21-127"><path d="M128 191.944c35.346 0 64-28.654 64-64 0-35.346-28.654-64-64-64-35.346 0-64 28.654-64 64 0 35.346 28.654 64 64 64Z" fill="#fff"/><path d="m183.427 160-55.368 95.899A127.802 127.802 0 0 0 238.739 64H128l-.092.055a63.91 63.91 0 0 1 55.529 31.903 63.911 63.911 0 0 1-.01 64.042Z" fill="url(#linear-gradient-21-127-a)"/><path d="M72.574 160 17.208 64.107a127.8 127.8 0 0 0 110.851 191.796L183.426 160v-.107a63.91 63.91 0 0 1-87.425 23.57A63.912 63.912 0 0 1 72.574 160Z" fill="url(#linear-gradient-21-127-b)"/><path d="M128 178.667c27.982 0 50.667-22.685 50.667-50.667 0-27.982-22.685-50.667-50.667-50.667-27.982 0-50.667 22.685-50.667 50.667 0 27.982 22.685 50.667 50.667 50.667Z" fill="#1A73E8"/><path d="M128 64h110.734a127.801 127.801 0 0 0-221.53.1L72.572 160l.094.053A63.917 63.917 0 0 1 128 64Z" fill="url(#linear-gradient-21-127-c)"/><defs><linearGradient id="linear-gradient-21-127-a" x1="114.542" y1="247.257" x2="224.212" y2="57.304" gradientUnits="userSpaceOnUse"><stop stop-color="#FCC934"/><stop offset="1" stop-color="#FBBC04"/></linearGradient><linearGradient id="linear-gradient-21-127-b" x1="150.604" y1="214.88" x2="-11.338" y2="121.383" gradientUnits="userSpaceOnUse"><stop stop-color="#1E8E3E"/><stop offset="1" stop-color="#34A853"/></linearGradient><linearGradient id="linear-gradient-21-127-c" x1="19.815" y1="80.026" x2="236.3" y2="80.026" gradientUnits="userSpaceOnUse"><stop stop-color="#D93025"/><stop offset="1" stop-color="#EA4335"/></linearGradient></defs></g><g class="conditional" id="g-128-plus"><g clip-path="url(#clip-path-128-plus)"><path d="M128 191.99c35.346 0 64-28.654 64-64 0-35.347-28.654-64-64-64-35.346 0-64 28.653-64 64 0 35.346 28.654 64 64 64Z" fill="#fff"/><path d="M128 0a128 128 0 1 0 0 256 128 128 0 0 0 0-256Zm0 192a64.002 64.002 0 0 1-62.77-76.486A63.999 63.999 0 0 1 192 128a64 64 0 0 1-64 64Z" fill="#1A73E8"/><path d="M128 64h110.833a127.969 127.969 0 0 0-221.685.034L72.553 160l.028-.005A63.985 63.985 0 0 1 128 64Z" fill="url(#linear-gradient-128-plus-a)"/><path d="M72.596 160.021 17.18 64.037a127.968 127.968 0 0 0 110.872 191.968l55.406-95.965-.019-.022a63.98 63.98 0 0 1-55.42 32.008 63.99 63.99 0 0 1-55.423-32.005Z" fill="url(#linear-gradient-128-plus-b)"/><path d="m183.447 160.021-55.417 95.985a127.963 127.963 0 0 0 110.852-64.002 127.963 127.963 0 0 0-.038-128H128.032l-.01.026a63.986 63.986 0 0 1 55.425 95.991Z" fill="url(#linear-gradient-128-plus-c)"/><path d="M128 180c28.719 0 52-23.281 52-52s-23.281-52-52-52-52 23.281-52 52 23.281 52 52 52Z" fill="#1A73E8"/></g><defs><linearGradient id="linear-gradient-128-plus-a" x1="17.148" y1="80" x2="238.833" y2="80" gradientUnits="userSpaceOnUse"><stop stop-color="#D93025"/><stop offset="1" stop-color="#EA4335"/></linearGradient><linearGradient id="linear-gradient-128-plus-b" x1="144.525" y1="252.607" x2="33.683" y2="60.622" gradientUnits="userSpaceOnUse"><stop stop-color="#1E8E3E"/><stop offset="1" stop-color="#34A853"/></linearGradient><linearGradient id="linear-gradient-128-plus-c" x1="110.213" y1="254.866" x2="221.056" y2="62.881" gradientUnits="userSpaceOnUse"><stop stop-color="#FCC934"/><stop offset="1" stop-color="#FBBC04"/></linearGradient><clipPath id="clip-path-128-plus"><path fill="#fff" d="M0 0h256v256H0z"/></clipPath></defs></g></svg>
```

**Pull request merged**